cmd/doc.go

```diff
@@ -51,30 +52,31 @@ var docCmd = &cobra.Command{
 	Hidden: true,
 	Short:  "Generates documentation for the tool in Markdown format",
 	Long:   "Generates documentation for the tool in Markdown format, and stores them in the designated location",
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
How are we supressing help string display on console if we return back with error ?